mautic/mautic
# Fix normalize value for boolean contact field

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
In some cases, If properties are set in order first value Yes, second No, normalize value work not as expected. This PR fixed it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Create boolean contact fields
3. Set properties in table to `a:2:{s:3:"yes";s:3:"Yes";s:2:"no";s:2:"No";}`
4. Create contact and set to Yes
5. Go to contact detail and see normalized value as No
6. After PR you should see Yes in contact detail




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11235"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
